sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/PubsubToBigqueryIT.java

```diff
@@ -57,7 +57,7 @@ public void testSimpleInsert() throws Exception {
             + "event_timestamp TIMESTAMP, \n"
             + "attributes MAP<VARCHAR, VARCHAR>, \n"
             + "payload ROW< \n"
-            + "             id INTEGER, \n"
```

Comment:
Note: Most of the type bugs here are around implicit casts between `INTEGER` and `BIGINT` that seem to not quite work right anymore.